packages/ui/src/components/primitives/Message.tsx

```diff
@@ -7,17 +7,28 @@ import { DiscordAvatar } from './DiscordAvatar';
 import { useIsUserInServer } from '~ui/utils/hooks';
 import { getSnowflakeUTCDate } from '~ui/utils/snowflake';
 import { cn } from '~ui/utils/styling';
-import * as AlertDialog from '@radix-ui/react-alert-dialog';
-import { LinkButton, DiscordIcon, CloseIcon } from './base';
+import * as Collapsible from '@radix-ui/react-collapsible';
+import { LinkButton, DiscordIcon, Heading, Button } from './base';
 import {
 	trackEvent,
 	messageWithDiscordAccountToAnalyticsData,
 } from '@answeroverflow/hooks';
 import { getDiscordURLForMessage } from '~ui/utils/discord';
+import {
+	Gallery,
+	type Image as ImageType,
+	type ThumbnailImageProps,
+} from 'react-grid-gallery';
+import { useEffect } from 'react';
+import Lightbox, { type Slide } from 'yet-another-react-lightbox';
+import { Zoom, Counter } from 'yet-another-react-lightbox/plugins';
+import 'yet-another-react-lightbox/styles.css';
+import { ChevronDownIcon, ChevronUpIcon } from '@heroicons/react/24/outline';
 
 // eslint-disable-next-line @typescript-eslint/naming-convention
 const MessageContext = createContext<{
 	message: APIMessageWithDiscordAccount;
+	collapseContent?: boolean;
```

Comment:
This should be the same as the props that the top level message component takes in

Reply:
Extract to type and reuse i.e MessageProps

Reply:
Fixed :+1: - hope that's what you meant :)